constellations/gemini-user/src/internship/internship.sass

```diff
@@ -0,0 +1,56 @@
+@import "../utils/constants.sass"
+
+#view-intern
+    font-family: $fontFamilyRoboto
+    width: 65%
+    margin-left: auto
+    margin-right: auto
```

Comment:
Just to adhere more to CSS ordering, put font-family below margin-right (look at the comment I left at Line 13- I saw this afterwards oops)

Reply:
Can you link the CSS ordering doc that we have in the `resources` folder plz? :) I think you made it.